### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by dicodiaz**
* Add React Router & Links
* Build Header, Books, Categories & NoMatch components

I took into account the following requirements:
- Initialise React app.
- Add React Redux (`npm install react-redux`).
- Structure your application files using a "feature folder" approach and use the [*ducks* pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. Your files/folder structure could look like this:
    ```    
    ...
    /src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
    ``` 
- The *building blocks* of your app should be set as re-usable components.
- Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two `<Route>`s and `<Link>`s for the app's navigation:
    - Books - the default view
        - Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet)
        - Should have a form for adding a book (no functionality yet)
    - Categories
        - Should display "Under construction" text only.
- Styling is not required at this point.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @dicodiaz, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        </div>
+        <div className="col-3">
+          <select className="col-3 form-select">
+            <option selected>Category</option>
+          </select>
+        </div>
+        <button type="submit" className="col-2">
+          ADD BOOK
+        </button>
+      </form>
```
Comment (mke2111) [inline src/components/Books.js]: - [x] Make this another separate component in a different file. You will only need to import it to the main page.
- [x] This component should render a form with inputs for both the title and author of the book. You're currently having only the title input. Add the author input to your form

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                  <div className="v-line px-0 mx-3" />
+                  <button type="button">Edit</button>
+                </div>
+              </div>
+              <div>
+                <div className="row row-cols-auto mx-0">
+                  <div className="Oval-2" />
+                  <div>
+                    <p>64%</p>
+                    <p>Completed</p>
```
Comment (mke2111) [inline src/components/Books.js]: - [x] You should have a BookList component that renders a list of books using the Book component, so this means you need to create both the BookList component and the Book component. Separate this file to enable these changes. Remember that the Book component should take the book data as a prop and renders the title and the author.

**Comment by dicodiaz** [comment]
Awesome suggestions, @mke2111, I applied them all. Thank you very much.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @dicodiaz, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**